src/pages/Admin/Charity/EditProfile/Slug.tsx

```diff
@@ -1,8 +1,13 @@
 import { Field } from "components/form";
 import { BASE_URL } from "constants/env";
+import { appRoutes } from "constants/routes";
+import { Link } from "react-router-dom";
 import { FV } from "./types";
 
-export default function Slug() {
+export default function Slug({ initSlug = "" }) {
+  const link =
```

Comment:
Once I saved a slug, I was not able to delete it, got error:
`"One or more parameter values are not valid. The update expression attempted to update a secondary index key to a value that is not supported. The AttributeValue for a key attribute cannot contain an empty string value."`

Is this expected?

Reply:
this is bug in the edit profile handler, I have set `slug` as pk in a GSI, perphaps instead of setting to `""`, attribute slug should be removed when web-app sets it to `""`